ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -1547,7 +1547,7 @@ def execute_expiring_check(check):
         tasks.uninstall_replica(self.master, self.replicas[0])
 
         # Store the current date to restore at the end of the test
-        now = datetime.utcnow()
+        now = datetime.now(tz=datetime.UTC)
```

Comment:
As the import is
```
from datetime import datetime, timedelta
```
you need to replace with
```
from datetime import datetime, timedelta, UTC
now = datetime.now(tz=UTC)
```

Reply:
Thank you, fixed.